### ISSUE TITLE
ImportError with sklearn 0.21

### ISSUE THREAD

**Issue by jhuguetn** [nipreps/mriqc#798]
I bumped into the following error running a baseline `mriqc` command (Python 3.7):
```
Node: workflow_enumerator.anatMRIQCT1w.ComputeIQMs.ComputeQI2
Working directory: /.../anatMRIQCT1w/ComputeIQMs/.../ComputeQI2

Node inputs:

air_msk = /.../ArtifactMask/sub-01_T1w_conformed_hat.nii
in_file = /.../sub-01_T1w_conformed.nii

Traceback (most recent call last):
  File "/.../nipype/pipeline/plugins/multiproc.py", line 69, in run_node
...
  File "/.../mriqc/classifier/sklearn/__init__.py", line 9, in <module>
    from .cv_nested import ModelAndGridSearchCV
  File "/.../mriqc/classifier/sklearn/cv_nested.py", line 29, in <module>
    from sklearn.model_selection._validation import (
ImportError: cannot import name 'logger' from 'sklearn.model_selection._validation' (/.../sklearn/model_selection/_validation.py)
```

At first glance, issue seems an incompatibility between mriqc and the latest version(s) of scikit-learn (>=0.2.1). Specifically, since commit https://github.com/scikit-learn/scikit-learn/commit/8edd9f9f1597094a1f8024f4d091f21e9c1fa66a, `sklearn/model_selection/_validation.py` script has had joblib's `logger` module import removed (see [here](https://github.com/scikit-learn/scikit-learn/commit/8edd9f9f1597094a1f8024f4d091f21e9c1fa66a#diff-60033c11a662f460e1567effd5faa6f0L26)) while it is still being imported and used by `mriqc`. See here:
https://github.com/poldracklab/mriqc/blob/255c8823992cd0a93680a1d062906989120c7733/mriqc/classifier/sklearn/cv_nested.py#L29-L31  
  
Note however that downgrading dependency package `sklearn` to 0.20.x works as a workaround.

**Comment by MousaMustafa** [comment]
Hi,
I am having the same issue, and im trying to install the sklearn package 0.20 but i am not being able to.
could you help in downgrading sklearn?
I have already tried uninstalling and then installing
pip uninstall sklearn
pip install -v sklearn==0.20.4
but it didnt work.

**Comment by munoztd0** [comment]
Same problme here

**Comment by oesteban** [comment]
The classifier has been stripped off from this package and moved to https://github.com/nipreps/mriqc-learn

Failure to keep up with new scikit learn versions has been addressed in new release 22.0.0rc1
